record-and-playback/presentation/playback/presentation/2.0/playback.js

```diff
@@ -289,235 +281,258 @@ generateThumbnails = function() {
   for (var i in imagesList) {
     $("#thumbnails").append(elementsMap[imagesList[i]]);
   }
-}
+};
 
-function checkUrl(url)
-{
-  console.log("==Checking Url",url);
-  var http = new XMLHttpRequest();
-  http.open('HEAD', url, false);
-  try {
-    http.send();
-  } catch (e) {
-    return false;
+function loadVideo() {
+  logger.info("==Loading video");
+  var video = document.createElement("video");
+  video.setAttribute('id','video');
+  video.setAttribute('class','webcam');
+  video.setAttribute('preload','auto');
+
+  let webmsource = document.createElement("source");
+  webmsource.setAttribute('src', url + '/video/webcams.webm');
+  webmsource.setAttribute('type','video/webm; codecs="vp8.0, vorbis"');
+  video.appendChild(webmsource);
+
+  let mp4source = document.createElement("source");
```

Comment:
I think leaving mp4 source here is OK, if mp4 is not present it will just give 404 and the browser will use the other format.

That said, I kind of wish we had a way of passing the list of formats from the processing to the playback, that would make it possible to switch to vp9 later.